packages/lib-classifier/src/plugins/tasks/models/Annotation.js

```diff
@@ -1,7 +1,8 @@
 import { types } from 'mobx-state-tree'
 
 const Annotation = types.model('Annotation', {
-  task: types.identifier,
+  id: types.identifier,
+  task: types.string,
```

Comment:
Note to self: confirm that the changes from` task:identifier` -> `id.identifier, task:string` are consistent

Reply:
👌